docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,172 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
 
+.. index:: pair: command groups;statistics command group
+
+.. _`Statistics command group`:
+
+``statistics`` command group
+----------------------------
+
+The ``statistics`` command group includes commands that manage and display
+statistics on the WBEM server operations executed by pywbemcli.
+
+Statistics can be gathered in the pywbemcli client itself and also separately
+by some WBEM servers.
+
+These servers provide statistical information on the overall server
+time that can be reported to the client with each operation response.  The
+enabling/disabling of client statistics and server statistics are separate.
+
+Client statistics gathering is enabled with the pywbemcli general option
+``--timestats`` / ``-T`` including  number of operations
+executed for each operation type, operation execution time (mininum, maximum, and
+average), minimum, maximum, and average size of requests and responses, and
+the server response time if it is enabled on the server.
+
+.. index:: pair: Operation statistics;statistics commands
+
+.. _Operation statistics:
+
+There are two statistics gathering capabilities:
+
+1. pywbem gathers statistics and pywbbemcli reports them on requests and
+responses executed by pywbemcli against a WBEM server. This statistics
+gathering is enabled with the general option ``--timestats`` / ``-T``.
+
```

Comment:
After finally understanding how "--timestats none" works, it became clear to me that it is not obvious how often the client maintains its client statistics:
* If used in command mode, the scope of its client statistics is only that single command, and the only way to see it is "--timestats auto".
* If used in interactive mode, the strange thing happens that interactive commands contribute to the client statistics of the next outer level, so the next outer level appears to maintain the client statistics.

The strangeness of "none" increases when looking at the statistics status in interactive mode and it says "none" but still keeps statistics.

**DISCUSSION**

Reply:
1. Clarify that once -T ... is executed statistics is always collected.
2. Client statistics lifecycle lives for the life of the interactive mode and only for the one command executed if in cmd mode. 
3. Question of exactly what is the lifecycle of statistics maintenance
4. Confirm and document the action if you change server in interactive mode.

This should be all documentation changes unless there is an issue with statistics and server change in interactive mode.

Reply:
DONE